runtime/compiler/control/J9Options.cpp

```diff
@@ -1735,13 +1735,15 @@ J9::Options::fePreProcess(void * base)
 
          UDATA largePageSize = 0;
          UDATA largePageFlags = 0;
-         int32_t lpArgIndex;
          UDATA lpSize = 0;
+         int32_t lpArgIndex = FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, "-Xlp", 0);
 
          // -Xlp<size>, attempt to use specified page size
-         if ((lpArgIndex=FIND_ARG_IN_VMARGS(EXACT_MEMORY_MATCH, "-Xlp", 0)) >= 0)
+         // GCC 7.3 on PPC64LE Bug FIND_ARG_IN_VMARGS can not be called directly inside the if conditional
```

Comment:
FYI the VM unofficial (and OMR?) coding standards don't allow assignment inside a if

Reply:
Then shall we create an issue to track the removal of such instances? Just those that relate to FIND_ARG_IN_VMARGS. 

It would help ensure that we don't run into this problem with any other option.

Reply:
can't hurt

Reply:
Done. See https://github.com/eclipse/openj9/issues/7746

Also removed comment
24db528392838ede4712c5caba786990d692a6cc

Reply:
Resolved.